src/pkg/skopeo/skopeo.go

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © 2023 Red Hat Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package skopeo
+
+import (
+	"bytes"
+	"encoding/json"
+
+	"github.com/containers/toolbox/pkg/shell"
+	"github.com/docker/go-units"
+)
+
+type Layer struct {
+	Size int64
+}
+type Image struct {
+	LayersData []Layer
+}
+
+func Inspect(target string) (string, error) {
+	var stdout bytes.Buffer
+	targetWithTransport := "docker://" + target
+
+	args := []string{"inspect", targetWithTransport}
+	if err := shell.Run("skopeo", nil, &stdout, nil, args...); err != nil {
+		return "", err
+	}
+
+	output := stdout.Bytes()
+	var image Image
+	if err := json.Unmarshal(output, &image); err != nil {
```

Comment:
Like we were discussing on chat, were you able to test the size calculation with a Skopeo older than `v1.10.0`?  That's when the LayersData array was [added](https://github.com/containers/skopeo/pull/1738) to `skopeo inspect` so, I am curious how it handles older versions.

Reply:
I tested with an older Skopeo and found that the prompt was:
```
Download <fully-qualified-image-name> [0B]? [y/N]:
```

I took the liberty to make the necessary changes to not show that `0B`, as we discussed [before](https://github.com/containers/toolbox/pull/1208#discussion_r1126577835).